Lib/lib2to3/__init__.py

```diff
@@ -1 +1,8 @@
-#empty
+import warnings
+
+
+warnings.warn(
+    "lib2to3 package is deprecated and may not be able to parse Python 3.10+",
+    PendingDeprecationWarning,
```

Comment:
I was under the impression that we don't use PendingDeprecationWarning any more: https://bugs.python.org/issue36404